README.md

```diff
@@ -28,19 +25,18 @@
 
 <a href="https://xn--931a.moe/"><img src="https://github.com/misskey-dev/misskey/blob/develop/assets/ai.png?raw=true" align="right" height="320px"/></a>
 
-## ✨ Features
+## ✨ Highlights
 - **ActivityPub support**\
-	It is possible to interact with other software.
+	Federates well with Mastodon, Pleroma and more.
 - **Reactions**\
-	You can add "reactions" to each post, making it easy for you to express your feelings.
+	Choose from a palette of reactions to express your feelings.
 - **Drive**\
 	An interface to manage uploaded files such as images, videos, sounds, etc.
 	You can also organize your favorite content into folders, making it easy to share again.
 - **Rich Web UI**\
 	Misskey has a rich WebUI by default.
-	It is highly customizable by flexibly changing the layout and installing various widgets and themes.
-	Furthermore, plug-ins can be created using AiScript, a original programming language.
-- and more...
+	It is highly customizable with different layouts, widgets and themes.
+	Furthermore, plug-ins can be created using AiScript, an original programming language.
```

Comment:
You can even add plugins with AiScript, a custom programming language for Misskey!